darwin/dataset/download_manager.py

```diff
@@ -103,11 +103,11 @@ def download_all_images_from_annotations(
             continue
 
         if not force_replace:
-            # Check collisions on image filename and json filename on the system
-            if annotation.filename in existing_images:
-                continue
-            if sanitize_filename(annotation_path.stem) in existing_images:
+            # Check the planned path for the image against the existing images
+            planned_image_path = images_path / Path(annotation.remote_path).relative_to('/') / Path(annotation.filename)
```

Comment:
`Path(something).relative_to('/')` is POSIX specific (windows may not like it) and should be equivalent to `Path(something).resolve().absolute()`

You need the `resolve` step to prevent it including incorrect symlinks or `..` segs that are unresolved.

Reply:
`Path(something).resolve().absolute()` doesn't strip the leading `/` in from `annotation.remote_path` which prevents concatenation, so instead I'll remove the leading `/` using string methods

Reply:
Just made a change to account for this